lib/formatter.js

```diff
@@ -104,19 +105,20 @@ Formatter.prototype.fail = function (fail) {
   )
   return output.join(LF)
 }
-
-Formatter.prototype.prettifyError = function (assertion) {
-  var rawError = assertion.error.raw
-  var ret = rawError.split(LF)
-  var stack = assertion.error.stack
-  if (stack) {
-    stack = stack.split(LF)
-    var padding = repeat(' ', ret[ret.length - 1].length)
-    ret = ret.concat(stack.map(function (s) {
-      return padding + s
-    }))
+if(process.argv[3]==1){
+  Formatter.prototype.prettifyError = function (assertion) {
+    var rawError = assertion.error.raw
+    var ret = rawError.split(LF)
+    var stack = assertion.error.stack
+    if (stack) {
+      stack = stack.split(LF)
+      var padding = repeat(' ', ret[ret.length - 1].length)
+      ret = ret.concat(stack.map(function (s) {
+        return padding 
+      }))
```

Comment:
This `map()` may not be useful anymore if you remove `+s`. This logic might need some revision. In fact, you may be able to remove this expression.